app/build.gradle

```diff
@@ -76,4 +77,6 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.7.0'
     implementation 'androidx.recyclerview:recyclerview:1.3.2'
     implementation 'androidx.media:media:1.7.0'
+    implementation 'com.google.android.material:material:1.12.0'
+    implementation 'androidx.preference:preference-ktx:1.2.1'
```

Comment:
could migrating the preferences perhaps be split from this? or is it an integral part?

Reply:
well they can probably be split. if necessary. I'll see to it and if possible create split pr with cherry picks from this.